backend/services/implementations/__tests__/statisticService.test.ts

```diff
@@ -103,6 +103,22 @@ describe("mongo statisticService", (): void => {
     expect(actualResult).toEqual(0);
   });
 
+  it("getMedianScoreByTest with multiple submissions", async () => {
```

Comment:
can you split this into two tests - one with even submissions and with odd submissions?

Reply:
Done!